src/main/blackbox/blackbox.h

```diff
@@ -29,7 +29,10 @@ typedef enum BlackboxDevice {
     BLACKBOX_DEVICE_NONE = 0,
     BLACKBOX_DEVICE_FLASH = 1,
     BLACKBOX_DEVICE_SDCARD = 2,
-    BLACKBOX_DEVICE_SERIAL = 3
+    BLACKBOX_DEVICE_SERIAL = 3,
+#ifdef USE_BLACKBOX_VIRTUAL
+    BLACKBOX_DEVICE_VIRTUAL = 4
+#endif
```

Comment:
Already approved - but think this one is not needed.
```suggestion
    BLACKBOX_DEVICE_VIRTUAL = 4
```